radix-engine-common/src/crypto/secp256k1/private_key.rs

```diff
@@ -1,20 +1,24 @@
 use crate::internal_prelude::*;
-use ::secp256k1::{Message, PublicKey, SecretKey, SECP256K1};
+use ::secp256k1::{All, Message, PublicKey, Secp256k1, SecretKey};
 
 use super::Secp256k1Signature;
 
+lazy_static::lazy_static! {
+    pub static ref SECP256K1_CTX: Secp256k1<All> = secp256k1::Secp256k1::new();
```

Comment:
Probably would want this to be a `pub(crate)`?

Also, is there any advantage to using this over the provided global context in the `secp256k1` crate?

Reply:
> Also, is there any advantage to using this over the provided global context in the `secp256k1` crate?
global context required `global-context` feature for `secp2561` which enforces 'std'